src/Parser/EnhancementShaman/Modules/ShamanCore/GCD.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class GCD extends Module {
+  suggestions(when) {
```

Comment:
This suggestion would be better placed in the Always Be Casting module where it's also calculated.